### PR TITLE
Issue 18016: Add infos count to `oc status`

### PR DESCRIPTION
**Pull request by damemi**
Fix for https://github.com/openshift/origin/issues/18016

Now `oc status` without `-v` will show a count for number of errors, warnings, and info identified. Also refactored this output to reduce redundant switch statements with including infos.

### CODE REVIEW

File: pkg/oc/cli/describe/projectstatus.go [hunk 1]
```diff
@@ -355,16 +355,26 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
...
+				markerString = fmt.Sprintf("%s%s", markerString, m)
+				count++
+			}
+		}
 
-		case !d.Suggest && len(warningMarkers) > 0:
-			fmt.Fprintf(out, "%s identified, use '%[2]s status -v' to see details.\n", warnings, d.CommandBaseName)
+		switch {
+		case !d.Suggest && ((len(errorMarkers) > 0 && errorSuggestions > 0) || len(warningMarkers) > 0 || len(infoMarkers) > 0):
+			fmt.Fprintf(out, "%s identified, use '%s status -v' to see details.\n", markerString, d.CommandBaseName)
```
Comment (juanvallejo) [inline pkg/oc/cli/describe/projectstatus.go]: I don't see where the count for each of these is printed.

Comment (damemi) [reply pkg/oc/cli/describe/projectstatus.go]: Sure, I wasn't sure if it grammatically made more sense to have, eg, "2 infos identified" vs "2 info identified"

Comment (damemi) [reply pkg/oc/cli/describe/projectstatus.go]: Updated

**Comment by juanvallejo** [comment]
Changes look good to me
Tagging @soltysh for approval

**Pull request merged**